src/components/HeroBlock/HeroBlock.jsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+import HeroBlockComponent from "./HeroBlockComponent";
+import HeroBlockPicture from "./img/heroBlockPicture.png";
+
+function HeroBlock() {
+  return (
+    <HeroBlockComponent
+      title="Izbaudi profesijas ēnošanas piedzīvojumu un atklāj savu karjeras
+  potenciālu!"
+      subtitle_span="TryWork"
+      subtitle="ir platforma, kurā motivētas personas satiekas ar atsaucīgiem
+  Ēnu devējiem."
+      button_left_text="Reģistrējies kā Ēnotājs"
+      button_left_link="/registration/register-shadower"
```

Comment:
Let's move this props to own file

Reply:
Should I create a file where is an object with values and then write the HeroBlock like this <Heroblock {...props} />?

Reply:
I't better to do as we did in footer, .config.js file